packages/app/src/pages/Specs/Runner.vue

```diff
@@ -1,22 +1,66 @@
 <template>
-  <SpecRunnerContainer
-    v-if="query.data.value?.currentProject?.specs"
+  <!--
+    Run Mode is a more minimal UI.
+    It does not render things like the SpecList,
+    Side and Top Nav, etc.
+    It also has no GraphQL dependency.
+  -->
+  <SpecRunnerContainerRunMode
+    v-if="isRunMode"
+    :run-mode-specs="specs"
+  />
+
+  <!--
+    Open Mode is the full Cypress runner UI -
+    including things like the SpecList,
+    Side and Top Nav, Selector Playgroundn etc.
+    It is driven by GraphQL and urql.
+  -->
+  <SpecRunnerContainerOpenMode
+    v-else-if="query.data.value?.currentProject?.specs"
     :gql="query.data.value"
   />
 </template>
 
 <script lang="ts" setup>
 import { gql, useQuery } from '@urql/vue'
 import { SpecPageContainerDocument } from '../../generated/graphql'
-import SpecRunnerContainer from '../../runner/SpecRunnerContainer.vue'
+import { getAutIframeModel } from '../../runner'
+import SpecRunnerContainerOpenMode from '../../runner/SpecRunnerContainerOpenMode.vue'
+import SpecRunnerContainerRunMode from '../../runner/SpecRunnerContainerRunMode.vue'
+import { togglePlayground } from '../../runner/utils'
 
 gql`
 query SpecPageContainer {
   ...SpecRunner
 }
 `
 
-const query = useQuery({ query: SpecPageContainerDocument })
+const isRunMode = window.__CYPRESS_MODE__ === 'run'
+
+// in run mode, we are not using GraphQL or urql
+// for performance - run mode does not need the
+// same level of runner interactivity as open mode.
+// by setting `cache-only` in run mode, urql will not trigger any
+// requests, which is what we want.
+const query = useQuery({
+  query: SpecPageContainerDocument,
+  requestPolicy: isRunMode && window.top === window ? 'cache-only' : 'cache-and-network',
```

Comment:
Should we replace this with `pause`? [docs](https://formidable.com/open-source/urql/docs/basics/react-preact/#pausing-usequery)

Reply:
nice idea, done (will push in w/ other fixes)